app/javascript/packages/verify-flow/verify-flow.tsx

```diff
@@ -113,8 +107,8 @@ function VerifyFlow({
   const [currentStep, setCurrentStep] = useState(steps[0].name);
   const [initialStep, setCompletedStep] = useInitialStepValidation(basePath, steps);
   const context = useMemo(
-    () => ({ startOverURL, cancelURL, currentStep, basePath, resetPasswordUrl }),
-    [startOverURL, cancelURL, currentStep, basePath, resetPasswordUrl],
+    () => ({ startOverURL, cancelURL, currentStep, basePath }),
+    [startOverURL, cancelURL, currentStep, basePath],
```

Comment:
would this be a good spot to use `useObjectMemo` from https://github.com/18F/identity-idp/pull/6363 ?

Reply:
> would this be a good spot to use `useObjectMemo` from #6363 ?

Yep, this context was updated in #6363. It wasn't merged at the time, but now is, so I'll rebase and update accordingly.

Reply:
Updated in rebased f920282ef.